giraffe/src/components/GeoLayer.tsx

```diff
@@ -122,17 +131,15 @@ const onAutoResize = (
     lonOnLastRender,
     zoomOnLastRender,
   } = lastRenderProperties
-
   updateQuery(
     props,
     lastRenderProperties,
     width,
     height,
-    latOnLastRender === null ? lat : latOnLastRender,
-    lonOnLastRender === null ? lon : lonOnLastRender,
-    zoomOnLastRender === null ? zoom : zoomOnLastRender
+    latOnLastRender === undefined ? lat : latOnLastRender,
```

Comment:
Can we clean this up a little bit? I don't think here you would need to have a condition to check for undefined. Instead you should be able to `latOnLastRender || lat` for what you're trying to pass in.

Reply:
The problem here is that lat, lon and zoom are numbers. The cleanup you propose would not work if `latOnLastRender === 0, lonOnLastRender === 0 or zoomOnLastRender ===0` I propose to keep the code as is.

Reply:
Here you are checking if `latOnLastRender === undefined`. If it is undefined it returns `lat` but if it will be `latOnLastRender = 0`, the condition for `undefined` is false and it returns `latOnLastRender` which is `0`. How is it different?

Reply:
Let's assume that `latOnLastRender === 0, lat === 100`. The current code returns 0. The shorthand you proposed `latOnLastRender || lat` returns '0 || lat`.  In our example it means `0 || 100` which is evaluated as `100` which is not equal to `0`, the correct result.